### PR TITLE
fixed little endian versions

### PR DESCRIPTION
**Pull request by extrawurst**
fixed copy paste bug of little endian versions

### CODE REVIEW

File: buffercursor.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
 var assert = require('assert');
 var util = require('util');
 
-var BufferCursor = module.exports = function(buff) {
+var BufferCursor = module.exports.BufferCursor = function(buff) {
```
Comment (tjfontaine) [inline buffercursor.js:24]: why is this necessary?

**Comment by tjfontaine** [comment]
Fixing the LE/BE typo is indeed necessary, and I'm not opposed to including the typescript, I'm just more fond of exporting this as the actual module

File: buffercursor.js (hunk 1 above)
Comment (extrawurst) [inline buffercursor.js:24]: this was necessary to make it work with typescript :(
